activities/migrations/0002_alter_post_visibility.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 4.1.3 on 2022-11-26 01:56
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("activities", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="post",
+            name="visibility",
+            field=models.IntegerField(
+                choices=[
+                    (0, "Public"),
+                    (4, "Local Only"),
```

Comment:
Since CHOICES doesn't affect the database, patch this into the previous migration instead if you could!